docs/dev/guides/publishing-bundles.md

```diff
@@ -59,19 +59,19 @@ If you do **not** add a _Contao Manager Plugin_ to your extension, it will still
 work with Contao, but:
 
  - it will require a user to manually register the bundle class in Symfony.
- - therefore the Contao Manager will list the package as **not being compatible**
-   to the Contao Managed Edition, and won't allow it's installation with
-   a push of the button.
+ - therefore, the Contao Manager will list the package as **not being compatible**
+   to the Contao Managed Edition, and won't allow its installation with
+   a push of a button.
 
 
 ## Publishing to the Contao ecosystem
 
-If you followed so far, your extension is now technically compatible with Contao, 
-but you also want Contao users to discover it. In most cases, this means having the package
-available in the Contao Manager search. A public version of this search index
+If you've been following along so far, your extension is now technically compatible with Contao 
```

Comment:
I think we should avoid contractions like that in the documentation, shouldn't we?

Reply:
Right

Reply:
I've updated the wording.